### ISSUE TITLE
A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan_Keyboard:The scroll bar is not accessible. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9266]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS2.1.1;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS2.1.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Navigate till next button & hit enter.
10. Provide all the details for Deploy Dacpac & hit next.
11. Navigate to the table using tab.

**Actual:**
On navigating on the page the tab focus after 3 is on the checkbox, the scroll bar is not accessible hence the data in the table will not be accessible to keyboard users.

**Expected:**
The scroll bar should be accessible after line 3 on the tab.

**User Impact:**
Keyboard users will be able to view the data in the table.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan_Keyboard9266.zip](https://github.com/microsoft/azuredatastudio/files/4322625/A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan_Keyboard9266.zip)

**Comment by alanrenmsft** [comment]
@kisantia , could you please see whether the table can be focused by keyboard? and use the voice over table navigation keys to navigate inside the table.

**Comment by alanrenmsft** [comment]
actually I remembered previously we have made changes to make sure readonly tables are not keyboard focusable.

**Comment by alanrenmsft** [comment]
I will ask the accessibility team for guideline

**Comment by kisantia** [comment]
@alanrenmsft did you hear back from the accessibility team on what we should do here? We previously had this issue #6729 where tab navigation wasn't supposed to be available when the table cells aren't actionable.

**Comment by udeeshagautam** [comment]
Chatted with @alanrenmsft - will allow tabbing in table. follow-up do we need move our with tab OR ctl+end + tab is enough.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.42.0
[9266_Fixed.webm](https://user-images.githubusercontent.com/85863999/218073896-f2d6a8f2-6a9b-4181-abff-0c8dfb9b6b9d.webm)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
